### PR TITLE
add new party map design

### PR DESCRIPTION
**Pull request by denisdimitrov**

### CODE REVIEW

File: src/components/templates/PartyMap/PartyMap.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.camp-container {
```
Comment (thecadams) [inline src/components/templates/PartyMap/PartyMap.scss:1]: nit: can we fix the name of this wrapper class?

File: src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+import { Modal } from "react-bootstrap";
+
+import { CampRoomData } from "types/CampRoomData";
```
Comment (thecadams) [inline src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx:5]: Can we rename this one to suit the new naming?

File: src/settings.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export const SPARKLEVERSE_LOGO_URL = "/sparkleverse-logo.png";
 export const SPARKLE_LOGO_URL = "/sparkle-header.png";
 export const MEMRISE_LOGO_URL = "/memrise-logo.png";
 export const BURN_START_UTC_SECONDS = 1598770800; // Sunday Aug 30th, 2020 (easy to change later)
-export const VENUE_CHAT_AGE_DAYS = 10;
+export const VENUE_CHAT_AGE_DAYS = 30;
```
Comment (thecadams) [inline src/settings.ts:19]: sensible change and seems like a good idea; just wondering what motivated it?

**Review by thecadams — APPROVED** [review]
lgtm - more great things on the way to removing the themecamp and playa/preplaya templates, great stuff.

File: src/settings.ts (hunk 3 above)
Comment (denisdimitrov) [reply src/settings.ts:19]: Ed, yesterday.

**Review by thecadams — APPROVED** [review]
lgtm!

File: src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx [hunk 4]
```diff
@@ -1,22 +1,18 @@
 import React from "react";
+import { formatUtcSeconds } from "utils/time";
+import { VenueEvent } from "types/VenueEvent";
 
-import { retainAttendance } from "store/actions/Attendance";
-import { formatMinute } from "utils/time";
-import { RoomEventData } from "types/RoomEventData";
 import { useDispatch } from "hooks/useDispatch";
-
-import "./ScheduleItem.scss";
```
Comment (0xdevalias) [inline src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx:8]: [TECH-DEBT] See https://github.com/sparkletown/internal-sparkle-issues/issues/344#issuecomment-777912917

**Pull request merged**